pkg/internal_registry/registry.go

```diff
@@ -0,0 +1,67 @@
+//
+// Copyright (c) 2019-2020 Red Hat, Inc.
+// This program and the accompanying materials are made
+// available under the terms of the Eclipse Public License 2.0
+// which is available at https://www.eclipse.org/legal/epl-2.0/
+//
+// SPDX-License-Identifier: EPL-2.0
+//
+// Contributors:
+//   Red Hat, Inc. - initial API and implementation
+//
+
+package registry
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+
+	"github.com/eclipse/che-plugin-broker/model"
+	brokerModel "github.com/eclipse/che-plugin-broker/model"
+	"gopkg.in/yaml.v1"
```

Comment:
There is also a `yaml.v2` API version that might be better, though I don't know the difference. Also this isn't in `go.mod` -- we need to run `go mod tidy` and commit `go.mod` and `go.sum`.